[2023-10-27 14:17:02,079] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:17:02,085] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:17:02,086] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:17:02,086] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:17:02,086] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:17:02,097] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 14:17:02,100] {standard_task_runner.py:52} INFO - Started process 17432 to run task
[2023-10-27 14:17:02,104] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp9k6yy00r', '--error-file', '/tmp/tmpnq8fad81']
[2023-10-27 14:17:02,110] {standard_task_runner.py:80} INFO - Job 27: Subtask GOOG
[2023-10-27 14:17:02,167] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:17:02,214] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 14:17:02,765] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 18, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 739, in get_handle
    check_parent_directory(str(handle))
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 604, in check_parent_directory
    raise OSError(rf"Cannot save file into a non-existent directory: '{parent}'")
OSError: Cannot save file into a non-existent directory: '/home/gabriel/Documents/project_finance/datalake/stocks/GOOG'
[2023-10-27 14:17:02,770] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T171702, end_date=20231027T171702
[2023-10-27 14:17:02,781] {standard_task_runner.py:92} ERROR - Failed to execute job 27 for task GOOG (Cannot save file into a non-existent directory: '/home/gabriel/Documents/project_finance/datalake/stocks/GOOG'; 17432)
[2023-10-27 14:17:02,820] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 14:17:02,835] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:30:31,909] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:30:31,921] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:30:31,922] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:30:31,922] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:30:31,922] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:30:31,951] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 14:30:31,969] {standard_task_runner.py:52} INFO - Started process 19774 to run task
[2023-10-27 14:30:31,986] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpsl8du1ip', '--error-file', '/tmp/tmpieszb6hj']
[2023-10-27 14:30:31,995] {standard_task_runner.py:80} INFO - Job 27: Subtask GOOG
[2023-10-27 14:30:32,063] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:30:32,191] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 14:30:32,741] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 20, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 863, in get_handle
    handle = open(
PermissionError: [Errno 13] Permission denied: '/GOOG_20231010.csv'
[2023-10-27 14:30:32,748] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T173031, end_date=20231027T173032
[2023-10-27 14:30:32,795] {standard_task_runner.py:92} ERROR - Failed to execute job 27 for task GOOG ([Errno 13] Permission denied: '/GOOG_20231010.csv'; 19774)
[2023-10-27 14:30:32,821] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 14:30:32,843] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:34:22,122] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:34:22,191] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:34:22,206] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:34:22,206] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:34:22,206] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:34:22,335] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 14:34:22,364] {standard_task_runner.py:52} INFO - Started process 20735 to run task
[2023-10-27 14:34:22,372] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpo3z_gr7d', '--error-file', '/tmp/tmpuio1dv2q']
[2023-10-27 14:34:22,384] {standard_task_runner.py:80} INFO - Job 27: Subtask GOOG
[2023-10-27 14:34:22,718] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:34:23,070] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 14:34:23,713] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 14:34:23,720] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T173422, end_date=20231027T173423
[2023-10-27 14:34:23,808] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 14:34:23,909] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:54:28,416] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:54:28,486] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 14:54:28,491] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:54:28,491] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:54:28,492] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:54:28,535] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 14:54:28,551] {standard_task_runner.py:52} INFO - Started process 23430 to run task
[2023-10-27 14:54:28,555] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpz4y_kgvw', '--error-file', '/tmp/tmpvv2figx2']
[2023-10-27 14:54:28,556] {standard_task_runner.py:80} INFO - Job 27: Subtask GOOG
[2023-10-27 14:54:28,649] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:54:28,725] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 14:54:29,318] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 14:54:29,331] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T175428, end_date=20231027T175429
[2023-10-27 14:54:29,377] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 14:54:29,399] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 15:23:39,839] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 15:23:39,857] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 15:23:39,857] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 15:23:39,857] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 15:23:39,857] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 15:23:39,874] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 15:23:39,884] {standard_task_runner.py:52} INFO - Started process 27279 to run task
[2023-10-27 15:23:39,894] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpcupz_og7', '--error-file', '/tmp/tmpu17p5dg2']
[2023-10-27 15:23:39,895] {standard_task_runner.py:80} INFO - Job 27: Subtask GOOG
[2023-10-27 15:23:40,031] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 15:23:40,078] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 15:23:40,488] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 15:23:40,492] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T182339, end_date=20231027T182340
[2023-10-27 15:23:40,514] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 15:23:40,528] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 17:02:08,657] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 17:02:08,664] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 17:02:08,664] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:02:08,664] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 17:02:08,664] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:02:08,673] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 17:02:08,677] {standard_task_runner.py:52} INFO - Started process 39497 to run task
[2023-10-27 17:02:08,680] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpp_5w4098', '--error-file', '/tmp/tmpwti2kcfb']
[2023-10-27 17:02:08,681] {standard_task_runner.py:80} INFO - Job 32: Subtask GOOG
[2023-10-27 17:02:08,715] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 17:02:08,785] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 17:02:09,281] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 20, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 863, in get_handle
    handle = open(
IsADirectoryError: [Errno 21] Is a directory: '/home/gabriel/Documents/project_finance/datalake/Bronze/stocks/GOOG'
[2023-10-27 17:02:09,304] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T200208, end_date=20231027T200209
[2023-10-27 17:02:09,334] {standard_task_runner.py:92} ERROR - Failed to execute job 32 for task GOOG ([Errno 21] Is a directory: '/home/gabriel/Documents/project_finance/datalake/Bronze/stocks/GOOG'; 39497)
[2023-10-27 17:02:09,355] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 17:02:09,480] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 17:09:26,173] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 17:09:26,179] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 17:09:26,180] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:09:26,180] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 17:09:26,180] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:09:26,193] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 17:09:26,198] {standard_task_runner.py:52} INFO - Started process 41216 to run task
[2023-10-27 17:09:26,203] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpzot1yve3', '--error-file', '/tmp/tmpys7nzf24']
[2023-10-27 17:09:26,204] {standard_task_runner.py:80} INFO - Job 32: Subtask GOOG
[2023-10-27 17:09:26,310] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 17:09:26,539] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 17:09:27,163] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 17:09:27,170] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T200926, end_date=20231027T200927
[2023-10-27 17:09:27,215] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 17:09:27,232] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 18:10:20,938] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 18:10:20,944] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [queued]>
[2023-10-27 18:10:20,945] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 18:10:20,945] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 18:10:20,945] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 18:10:20,957] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-10 00:00:00+00:00
[2023-10-27 18:10:20,961] {standard_task_runner.py:52} INFO - Started process 49366 to run task
[2023-10-27 18:10:20,964] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-10T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpk_ub0qab', '--error-file', '/tmp/tmpxlqzufm7']
[2023-10-27 18:10:20,965] {standard_task_runner.py:80} INFO - Job 31: Subtask GOOG
[2023-10-27 18:10:21,017] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-10T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 18:10:21,163] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-10T00:00:00+00:00
[2023-10-27 18:10:21,619] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 18:10:21,625] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231010T000000, start_date=20231027T211020, end_date=20231027T211021
[2023-10-27 18:10:21,653] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 18:10:21,688] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
